libc/unistd_getopt.c

```diff
@@ -0,0 +1,337 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libc-tests
+ *
+ * Testing POSIX compatibility of getopt() unistd.h function
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Marek Bialowas
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <ctype.h>
+
+#include <unity_fixture.h>
+
+
+enum getopterr { NOERR = 0,
+	OPTREQ = 1,
+	UNKNOWNOPT = 2,
+	UNKNOWNCHAR = 4,
+	NOARG = 8,
+	ABORT = 16 };
```

Comment:
Use defines for bitmask. Also, enums are lowercase in out coding style

Reply:
Switched for `#define`s. Thanks!